Sources/Subs-Db-postgresql.php

```diff
@@ -202,6 +202,13 @@ function smf_db_replacement__callback($matches)
 				smf_db_error_backtrace('Wrong value type sent to the database. Date expected. (' . $matches[2] . ')', '', E_USER_ERROR, __FILE__, __LINE__);
 		break;
 
+		case 'time':
+			if (preg_match('~^([0-1]?\d|2[0-3]):([0-5]\d):([0-5]\d)$~', $replacement, $time_matches) === 1)
+				return sprintf('\'%02d:%02d:%02d\'', $time_matches[1], $time_matches[2], $time_matches[3]);
```

Comment:
Well, according to [this](https://www.postgresql.org/docs/9.5/static/datatype-datetime.html#DATATYPE-DATETIME-INPUT), it is perfectly fine to give date and time literal input.

If you still think it is a problem, you will probably also want to make similar changes for all the other data types that `smf_db_replacement__callback()` handles as well. If so, it would make more sense to implement all those changes at once in a separate PR. 

But as I said, I really don't see the need. The current system already does things in a valid way.